src/predicates/intersects.jl

```diff
@@ -33,6 +33,20 @@ intersects(r::Ray, t::Triangle) = !isnothing(r ∩ t)
 
 intersects(t::Triangle, r::Ray) = intersects(r, t)
 
+function intersects(r::Ray, s::Sphere)
+  s₀ = center(s) - r(0)
+  h = hypot(s₀...)
+  radius(s) > h && return true
+  r₀ = r(1) - r(0)
+  abs(∠(s₀, r₀)) < asin(radius(s) / h)
```

Comment:
```suggestion
  u = center(s) - r(0)
  h = norm(u)
  radius(s) > h && return true
  v = r(1) - r(0)
  abs(∠(u, v)) < asin(radius(s) / h)
```
Can you please double check this suggested change?

Reply:
Looks good to me 👍